### PR TITLE
Fixes #1727 - Add PUT/PATCH/DELETE for zaakeigenschappen

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes https://github.com/VNG-Realisatie/gemma-zaken/issues/1727

### CODE REVIEW

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
There's a lot of stuff related to 1.1 features that snuck in the API schema, did you base this branch of master rather than `stable/1.0.x` by any change?

**Comment by SilviaAmAm** [comment]
As far as I can see, I based on stable/1.0.x :thinking: 
If I `git log`, then the last commit before the ones I added is:
```
commit 35210a923b9e9265fe5647b3c2334fccfeca1096 (upstream/stable/1.0.x)
Merge: b2c0975 f02b644
Author: Steven Bal 
Date:   Mon Jan 11 10:12:51 2021 +0100

    Merge pull request #193 from VNG-Realisatie/issue/fix-ci
    
    :green_heart: Add DEPLOY_BOT_TOKEN to docker push workflow
```

**Review by sergei-maertens — APPROVED** [review]

**Comment by sergei-maertens** [comment]
Closing to re-open against the master branch so that this could go in a 1.2 release.

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
I'm missing the right notifications mixin so that notifications are also emitted on (partial) update and destroy of zaakeigenschappen.

File: src/zrc/api/serializers/core.py [hunk 1]
```diff
@@ -820,6 +820,13 @@ def save(self, **kwargs):
 
 
 class ZaakEigenschapSerializer(NestedHyperlinkedModelSerializer):
+    zaak = serializers.HyperlinkedRelatedField(
+        queryset=Zaak.objects.all(),
+        view_name="zaak-detail",
+        lookup_field="uuid",
+        validators=[IsImmutableValidator()],
+    )
+
```
Comment (sergei-maertens) [inline src/zrc/api/serializers/core.py:829]: wouldn't we have achieved the same by specifying `"validators": [IsImmutableValidator()]` in the `extra_kwargs` for `zaak`?

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**